lib/cf/CloudControllerClient.js

```diff
@@ -275,6 +278,27 @@ class CloudControllerClient extends HttpClient {
     return this.getResource(`/v2/organizations/${org_guid}`, options);
   }
 
+  getMultiTenantServiceInstanceIdWithName(instance_name, org_guid, space_guid, service_name, options) {
+    return this.getResource(`/v2/spaces/${space_guid}/service_instances?q=name:${instance_name}`, options)
```

Comment:
can you instead use this [cc api](https://apidocs.cloudfoundry.org/280/service_instances/list_all_service_instances.html)  This API is generic enough to list all instances in any org or space. So you can build the query `q` based on what is present in the options. By doing so you would make this DAO method generic and truly reusable for any other usecases as well.